quizapp/src/main/java/billennium/quizapp/controller/ControllerConstants.java

```diff
@@ -6,6 +6,7 @@
     public static final String ID_PARAM = "/{id}";
     public static final String CANDIDATE = "/candidate";
     public static final String SAVE_CANDIDATE = "/saveCandidate";
+    public static final String GET_CANDIDATE = "/getCandidate";
```

Comment:
widzę tutaj błąd nazewnictwa path w REST. Proszę o sprawdzenie konwencji nazw dla REST  i poprawe. `SAVE_CANDIDATE ` też jest źle. Proszę o poprawę wszystkich w oddzielnym commicie we współpracy z @CzajkowskiArtur

Reply:
ok poprawimy rozumiem że chodzi o nazwnictwo: save-candidate i get-candidate  taki trochę kebab case

Reply:
https://restfulapi.net/resource-naming/

Reply:
`candidate/{id}` <- wyswietlenie